### DISCUSSION TITLE
arc init support for macros

### DISCUSSION THREAD

**Discussion by m-butler** [architect/architect#1254] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
After adding `@macros` followed by a macro file name in my .arc file, running `arc init` does not create the macros dir and associated macro js file.

**Describe the solution you'd like**
Create the macros dir under src as well as a js handler for the specified name (just like with other .arc declarations).

**Describe alternatives you've considered**
Creating the dir and js file manually.

**Additional context or notes**
@brianleroux asked that I enter this issue per a convo on slack.

**Comment by ryanblock** [comment]
This is a dope idea!
